src/components/Composer.vue

```diff
@@ -462,6 +461,21 @@ export default {
 		allRecipients() {
 			this.checkRecipientsKeys()
 		},
+		aliases(newAliases) {
+			console.debug('aliases changed')
+			if (this.selectedAlias === NO_ALIAS_SET) {
+				return
+			}
+
+			const newAlias = newAliases.find(alias => alias.id === this.selectedAlias.id && alias.aliasId === this.selectedAlias.aliasId)
+			if (newAlias === undefined) {
+				// selected alias does not exist anymore.
+				this.onAliasChange(newAliases[0])
```

Comment:
I wonder if newAliases could be empty. We have the default "alias" from the email address.

Reply:
we can defensively add a check

Reply:
should be fine without. the account is always there.